cmd/ui/src/views/QA/QA.tsx

```diff
@@ -60,10 +60,11 @@ const QualityAssurance: React.FC = () => {
 
     if (!contextType || (!contextId && (contextType === 'active-directory' || contextType === 'azure'))) {
         return (
-            <ContentPage
-                title='Data Quality'
-                data-testid='data-quality'
-                actionButton={
```

Comment:
Since we are no longer displaying the data selector in a position inline with the header we should remove this actionButton prop from the ContentPage component as it is no longer being used. We would also want to remove the wrapping flex container from the ContentPage and instead have only the Typography h1 element and the children within the encapsulating div.